### PR TITLE
Include Wikipedia link for molecules if one is provided

### PR DESCRIPTION
**Pull request by bnmajor**
Signed-off-by: Brianna Major <brianna.major@kitware.com>

### CODE REVIEW

**Comment by bnmajor** [comment]
The sample molecules that will be available for users to ingest will include a link to the Wikipedia page for that molecule. This adds the ability to display that link when it is provided.

File: src/components/molecule.js [hunk 1]
```diff
@@ -75,6 +75,13 @@ class Molecule extends Component {
     }
   }
 
+  formatLink = (props) => {
+    if (!isNil(props.name)) {
+      return <a href src={props.wikipediaUrl}>{props.name}</a>
+    }
+    return <a href src={props.wikipediaUrl}>Wikipedia Page</a>
+  }
+
```
Comment (cjh1) [inline src/components/molecule.js:83]: Where is this used?

Comment (bnmajor) [reply src/components/molecule.js:83]: In the `Molecule Properties` card on the molecule page the Wikipedia link is displayed if it is provided. The link text should be the molecule name, but if for some reason no common name has been provided the text will default to being 'Wikipedia Page'.

**Review by cjh1 — APPROVED** [review]
LGTM

File: src/components/molecule.js (hunk 1 above)
Comment (bnmajor) [reply src/components/molecule.js:83]: Sorry, misunderstood the question. The function was not being called, but should have been. This has been fixed now.

**Pull request merged**